docs/lib/analytics/existing-resources.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Use existing AWS resources
+description: Configure the Amplify Libraries to use existing AWS resources by referencing them in your configuration.
```

Comment:
nit: "use existing AWS resources (Pinpoint, Kinesis)". Probably makes sense to have the actual service lists out inside the description. Might be helpful with SEO.